SnapWar/ViewControllers/SWBattleViewController.swift

```diff
@@ -1,24 +1,56 @@
 import UIKit
 
 class SWBattleViewController: UIViewController {
-    let fullScreenImageView = SWFullscreenImageView(named: "test-image-2")
+    let fullScreenImageView = SWFullscreenImageView(named: "test-image-4")
+    let previewContainer = UIStackView()
     
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        view?.backgroundColor = UIColor.red
-        addTestImage()
+        view?.backgroundColor = SWColorType.white
+//        addTestImage()
+        addPreviewImage()
     }
     
     func addTestImage() {
         view.addSubview(fullScreenImageView)
         
     }
     
+    func addPreviewImage() {
+        view.addSubview(previewContainer)
+
+        previewContainer.axis = .vertical
+        previewContainer.alignment = .fill
+        previewContainer.distribution = .fillEqually
+        
+        previewContainer.translatesAutoresizingMaskIntoConstraints = false
+        previewContainer.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor).isActive = true
+        previewContainer.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor).isActive = true
+        previewContainer.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor).isActive = true
+        previewContainer.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor).isActive = true
+
+        let testPreviews = [
+            "test-image-1",
+            "test-image-3",
+        ]
+
+        for previewName in testPreviews {
+            let preview = SWImagePreview(named: previewName, frame: .zero)
+            previewContainer.addArrangedSubview(preview)
+        }
+        
+    }
+    
     override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
         fullScreenImageView.handleResize()
+        
+        let orientation = UIApplication.shared.statusBarOrientation
```

Comment:
i think `.statusBarOrientation` is actually deprecated for iOS 13 and Swift 5+.

this is how i think it should be done (which is also backwards compatible too):

```
extension UIWindow {
    static var isLandscape: Bool {
        if #available(iOS 13.0, *) {
            return UIApplication.shared.windows
                .first?
                .windowScene?
                .interfaceOrientation
                .isLandscape ?? false
        } else {
            return UIApplication.shared.statusBarOrientation.isLandscape
        }
    }
}

if (UIWindow.isLandscape()) {
...
} else {
...
}
```

Reply:
it might be useful to put the extension in another common file so that we can all access it!